internal/tools/listdir.go

```diff
@@ -20,3 +22,13 @@ func ListFiles(dir, extension string) ([]string, error) {
 	}
 	return res, nil
 }
+
+func AssertFolderRWXPermissions(name, dir string) {
+	fi, err := os.Lstat(dir)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if fi.Mode().Perm()&0750 != 0750 {
```

Comment:
what are you testing for exactly? rwx rx -

maybe we should instead use https://pkg.go.dev/golang.org/x/sys/unix#Access (see https://man7.org/linux/man-pages/man2/access.2.html for doku)

Reply:
Yes, that's better.